sdk/python/feast/client.py

```diff
@@ -655,10 +661,10 @@ def get_batch_features(
     def get_online_features(
         self,
         feature_refs: List[str],
-        entity_rows: List[GetOnlineFeaturesRequest.EntityRow],
+        entity_rows: List[Union[GetOnlineFeaturesRequest.EntityRow, Dict[str, Any]]],
         project: Optional[str] = None,
         omit_entities: bool = False,
-    ) -> GetOnlineFeaturesResponse:
+    ) -> OnlineResponse:
```

Comment:
Can you please add an example of retrieval here. Have a look here.https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html